xbmc/video/PlayerController.cpp

```diff
@@ -524,12 +511,17 @@ void CPlayerController::OnSliderChange(void *data, CGUISliderControl *slider)
 
   if (m_sliderAction == ACTION_ZOOM_OUT || m_sliderAction == ACTION_ZOOM_IN ||
       m_sliderAction == ACTION_INCREASE_PAR || m_sliderAction == ACTION_DECREASE_PAR ||
-      m_sliderAction == ACTION_VSHIFT_UP || m_sliderAction == ACTION_VSHIFT_DOWN ||
-      m_sliderAction == ACTION_SUBTITLE_VSHIFT_UP || m_sliderAction == ACTION_SUBTITLE_VSHIFT_DOWN)
+      m_sliderAction == ACTION_VSHIFT_UP || m_sliderAction == ACTION_VSHIFT_DOWN)
   {
     std::string strValue = StringUtils::Format("{:1.2f}", slider->GetFloatValue());
     slider->SetTextValue(strValue);
   }
+  else if (m_sliderAction == ACTION_SUBTITLE_VSHIFT_UP ||
+           m_sliderAction == ACTION_SUBTITLE_VSHIFT_DOWN)
+  {
+    std::string strValue = StringUtils::Format("{:d}px", static_cast<int>(slider->GetFloatValue()));
```

Comment:
```suggestion
    std::string strValue = StringUtils::Format("{:.0f}px", slider->GetFloatValue());
```

Reply:
here it is intended to avoid decimals

Reply:
Yep, and you can avoid the cast

Reply:
Just an FYI after testing this it seems fmt will round the value rather than truncate. It's probably fine but worth pointing out: https://godbolt.org/z/Ec8xrscej

basically
```c++

  float value = 1234.5678;

  fmt::print("float: {:.0f}\n", value);
  fmt::print("int  : {}\n", static_cast<int>(value));
```

output:
```
float: 1235
int  : 1234
```

Reply:
I think it is ok, this value has differents conversions/scale/changes from us and also in the libass library, i don't think it matters much it's just for general user reference